test/travis-before-install.sh

```diff
@@ -19,8 +19,15 @@ travis_retry go get \
   github.com/golang/lint/golint \
   github.com/mattn/goveralls \
   github.com/modocache/gover \
-  github.com/jcjones/github-pr-status \
-  github.com/letsencrypt/goose/cmd/goose
+  github.com/jcjones/github-pr-status &
+
+(wget https://github.com/jsha/boulder-tools/raw/master/goose.gz &&
```

Comment:
Can you toss a sha256sum check into this, to catch the event that a sophisticated adversary replaces your goose.gz with a more exciting version?